ring_doorbell/__init__.py

```diff
@@ -37,164 +22,19 @@ class Ring(object):
     def __init__(self, username, password,
                  auth_callback=None,
```

Comment:
We should rename `auth_callback` to `otp_callback` (One Time Password), as that is what it's for.